presto-parser/src/main/antlr4/com/facebook/presto/sql/parser/SqlBase.g4

```diff
@@ -60,7 +60,7 @@ statement
         (privilege (',' privilege)* | ALL PRIVILEGES)
         ON TABLE? qualifiedName FROM grantee=identifier                #revoke
     | SHOW GRANTS
-        ON (ALL | TABLE? qualifiedName)                                #showGrants
```

Comment:
It would be weird for TABLE to apply both to tables and views given that we distinguish between them in other contexts (SHOW CREATE TABLE vs SHOW CREATE VIEW, DROP TABLE vs DROP VIEW)